addons/explosives/functions/fnc_selectTrigger.sqf

```diff
@@ -22,6 +22,12 @@ TRACE_3("params",_explosive,_magazine,_trigger);
 
 private _config = ConfigFile >> "ACE_Triggers" >> _trigger;
 
+// Make selected trigger the active one (for keybind) if it's the first to be connected
+private _activeTrigger = GVAR(activeTrigger);
+if (_activeTrigger isEqualTo "" && {_trigger in ["Command", "MK16_Transmitter", "DeadManSwitch"]}) then {
```

Comment:
Not a fan of these checks that are case sensitive.

Reply:
Are you referring to the `in` check?
I tried to make it config based and just check if `_trigger` is a child class of `Command`, but `isKindOf` or `typeOf` just wouldn't work as expected. Any ideas?

Reply:
How is making case-sensitive checks non-sensitive better if the casing is always known ?